.github/ISSUE_TEMPLATE/feature.yml

```diff
@@ -0,0 +1,18 @@
+---
+# https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema
+name: Feature
+description: Some feature is not implemented yet
+assignees: ["ferretdb-bot"]
+labels: ["code/feature"]
+body:
```

Comment:
Here I'd also add "why", it will help us understand the need and prioritize the task.

Reply:
#1160